src/generator.rs

```diff
@@ -47,20 +46,54 @@ impl Generator {
         match self {
             Generator::Tonic => {
                 tonic_build::configure()
+                    .out_dir(output_directory) // If env var OUT_DIR is not set, Tonic throws a rather unhelpful error
                     .build_client(true)
                     .build_server(true)
                     .build_transport(true)
                     .include_file(Self::TONIC_INCLUDE_FILE)
                     .compile(&protos, includes)?;
             }
+            Generator::Protoc => {
+                let mut protoc_cmd = std::process::Command::new(protoc);
+
+                match language {
+                    Language::Python => {
+                        protoc_cmd.arg("--python_out").arg(output_directory);
+                    }
+                    Language::Rust => {
+                        panic!("Protoc cannot output Rust, use Generator::Tonic instead")
```

Comment:
never panic in production ready programs!

you can see that we return a `eyre::Result<()>` to indicate that this operation is failible – you can use `eyre::bail`